.github/ISSUE_TEMPLATE/BUG.yaml

```diff
@@ -0,0 +1,103 @@
+name: "Bug Report"
+description: Create an issue for a bug.
+title: "Bug: <title>"
+labels: ["bug", "triage required"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: "Description"
+      description: Please enter an description of the bug you are encountering
+      placeholder:
+    validations:
+      required: true
+  - type: input
```

Comment:
This is seriously over complicated i think. Just a simple test aread for a description/code etc. and an optional field to upload stuff is enough. 

IF the user does not need to go into all of these, fine, but I am really averse to repositories that ask me to fill complex forms to file an issue (as a user myself).